### PR TITLE
docs: update badge links

### PR DESCRIPTION
**Pull request by ericdeansanchez**

### CODE REVIEW

**Review by sherwinski — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -5,10 +5,10 @@
 `@imgix/js-core` is a JavaScript library for generating image URLs with [imgix](https://www.imgix.com/) that can be used in browser or server-side settings.
 
 [![NPM Version](https://img.shields.io/npm/v/imgix-core-js.svg)](https://www.npmjs.com/package/@imgix/js-core)
```
Comment (frederickfogerty) [inline README.md:7]: Should be `[NPM Version](https://www.npmjs.com/package/@imgix/js-core)`

File: README.md [hunk 2]
```diff
@@ -5,10 +5,10 @@
...
-[![Build Status](https://travis-ci.com/imgix/imgix-core-js.svg?branch=main)](https://travis-ci.com/github/imgix/js-core)
+[![Build Status](https://api.travis-ci.com/imgix/js-core.svg?branch=main)](https://travis-ci.com/github/imgix/js-core)
 [![Monthly Downloads](https://img.shields.io/npm/dm/imgix-core-js.svg)](https://www.npmjs.com/package/@imgix/js-core)
```
Comment (frederickfogerty) [inline README.md:9]: Should be `[Monthly Downloads](https://www.npmjs.com/package/@imgix/js-core)`

File: README.md [hunk 3]
```diff
@@ -5,10 +5,10 @@
...
 [![Minified Size](https://img.shields.io/bundlephobia/min/imgix-core-js)](https://bundlephobia.com/result?p=imgix-core-js)
```
Comment (frederickfogerty) [inline README.md:10]: Should be `[Minified Size](https://bundlephobia.com/result?p=@imgix/js-core)`

File: README.md [hunk 4]
```diff
@@ -5,10 +5,10 @@
...
-[![License](https://img.shields.io/github/license/imgix/imgix-core-js)](https://github.com/imgix/js-core/blob/main/LICENSE.md)
+[![License](https://img.shields.io/github/license/imgix/js-core)](https://github.com/imgix/js-core/blob/main/LICENSE.md)
 [![FOSSA Status](https://app.fossa.com/api/projects/git%2Bgithub.com%2Fimgix%2Fimgix-core-js.svg?type=shield)](https://app.fossa.com/projects/git%2Bgithub.com%2Fimgix%2Fimgix-core-js?ref=badge_shield)
```
Comment (frederickfogerty) [inline README.md:12]: I think we might have to re-import this project into FOSSA to update this URL

**Review by frederickfogerty — COMMENTED** [review]
Some more updates to be done here

**Pull request merged**